bot/exts/evergreen/githubinfo.py

```diff
@@ -72,26 +79,87 @@ async def get_github_info(self, ctx: commands.Context, username: Optional[str])
+                )
+
+            embed.add_field(
+                name="Public repos",
+                value=f"[{user_data['public_repos']}]({user_data['html_url']}?tab=repositories)"
+            )
 
             if user_data['type'] == "User":
                 embed.add_field(name="Gists", value=f"[{gists}](https://gist.github.com/{username})")
 
-                embed.add_field(name=f"Organization{'s' if len(orgs)!=1 else ''}",
-                                value=orgs_to_add if orgs else "No organizations")
-                embed.add_field(name="\u200b", value="\u200b")
+                embed.add_field(
+                    name=f"Organization{'s' if len(orgs)!=1 else ''}",
+                    value=orgs_to_add if orgs else "No organizations"
+                )
             embed.add_field(name="Website", value=blog)
 
         await ctx.send(embed=embed)
 
+    @github_group.command(name='repository', aliases=('repo',))
+    async def github_repo_info(self, ctx: commands.Context, repo: str) -> None:
+        """
+        Fetches a repositories' GitHub information. The repository should look like `user/reponame`.
+
+        Repository is optional and sends the help command if not specified.
+        """
+        async with ctx.typing():
+            repo_data = await self.fetch_data(f"{GITHUB_API_URL}/repos/{repo}")
+
+            # There won't be a message key if this repo exists
+            if repo_data.get('message') is not None:
+                embed = discord.Embed(
+                    title=random.choice(NEGATIVE_REPLIES),
+                    description="The requested repository was not found.",
+                    colour=Colours.soft_red
+                )
+
+                await ctx.send(embed=embed)
+                return
+
+        repo_owner = repo_data['owner']
+
+        parent = repo_data.get('parent')
+
+        embed = discord.Embed(
+            title=f"{repo_data['name']}",
+            description=repo_data["description"],
+            colour=discord.Colour.blurple(),
+            url=repo_data['html_url']
+        )
+
+        # If it's a fork, then it will have a parent key
+        if parent:
+            embed.description += f"\n\nForked from [{parent['full_name']}]({parent['html_url']})"
+
+        embed.set_author(
+            name=repo_owner["login"],
+            url=repo_owner["html_url"],
+            icon_url=repo_owner["avatar_url"]
+        )
+
+        repo_created_at = datetime.strptime(repo_data['created_at'], "%Y-%m-%dT%H:%M:%SZ").strftime("%d/%m/%Y")
+        last_pushed = datetime.strptime(repo_data['pushed_at'], "%Y-%m-%dT%H:%M:%SZ").strftime("%d/%m/%Y at %H:%M")
+
+        embed.set_footer(
+            text=(
+                f"{repo_data['forks_count']} ⑂ "
+                f"• {repo_data['stargazers_count']} ⭐ "
+                f"• Created At {repo_created_at} "
+                f"• Last Commit {last_pushed}"
+            )
+        )
```

Comment:
Could do:

You can set the `icon_url` (The URL of the footer icon) as the URL of the GitHub avatar of the owner

or you could even do this in the `discord.Embed()` (`set_author`).

Reply:
That's what I did.

Reply:
Oh yes, I totally missed that.